allocation/models/strategy.py

```diff
@@ -44,10 +44,10 @@ def get_instance_list(self, identity, limit_instances=[], limit_history=[]):
                 continue
             try:
                 allocation_instance = AllocInstance.from_core(
-                        inst,
-                        self.counting_behavior.start_date,
-                        limit_history=limit_history
-                    )
+                    inst,
+                    self.counting_behavior.start_date,
+                    limit_history=limit_history
+                )
```

Comment:
No, we are not ignoring E124.  I suspect that this is a case where the standard was contorted to make it easy for automated tools to check the formatting given some not so intuitive results.  Here is the test code that I am considering:

    # Just a test case to consider E124
   
   
    def funca(p1, p2, p3):
        return p1 + p2 + p3
   
   
    def funcb():
        a1 = 1
        a2 = 1
        a3 = 1
        # These do not yeild E124
        funca(
            a1,
            a2,
            a3
        )
        funca(
            a1,
            a2,
            a3
            )
        funca(a1,
              a2,
              a3)
        funca(funca(
            a1, a2, a3))
        funca(funca(
            a1,
            a2, 
            a3))
        funca(funca(
                a1,
                a2,
                a3
            ),  
            funca(a1, a2, a3),
            a3
        )
        funca( 
                a1, a2, a3
            )    
        funca(
                a1, a2, a3
        )       
      
        # Yields E124
        funca(a1,
              a2,
              a3
        )     
        funca(a1,
              a2,
              a3
            )
        funca(a1, a2, a3
        )
        funca(a1,
              a2,
              a3
             )